### PR TITLE
feat: refactor into npm package

### PR DESCRIPTION
**Pull request by mjmaurer**
Includes
- tests that don't depend on live airtable
- scheduling built in to change detector

Still need do a bit more testing. I'll try it out with Crown Heights staging

### CODE REVIEW

**Comment by mjmaurer** [comment]
@btmoore4

**Comment by mjmaurer** [comment]
Tested this package in staging and looks to be working fine

**Comment by mjmaurer** [comment]
Here's a reference implementation: https://github.com/crownheightsaid/mutual-aid-app/pull/39/files

File: src/ChangeDetector.js [hunk 1]
```diff
@@ -56,9 +85,12 @@ class ChangeDetector {
    *   record.getMeta() // returns the parsed meta for the record: {"lastValues":{...}}
    *   record.didChange("field name ") // returns true if the field changed (or is new) between the last observation and now
    */
-  async poll() {
+  async pollOnce() {
     const toExamine = await this.getModifiedRecords();
-    const recordsWithFieldChanges = toExamine.filter(this.hasFieldChanges);
+    const recordsWithFieldChanges = toExamine.filter(
```
Comment (alexquick) [inline src/ChangeDetector.js:90]: fat-arrow may be more idiomatic here and in similar cases?

File: src/ChangeDetector.test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  sinon.restore();
+});
+
+describe("Airtable Changes", () => {
+  describe("#pollOnce()", () => {
+    it("Should return records that are new", async () => {
+      const modifiedTime = "a date";
+      const base = new Airtable.Base("airtable", "baseId");
+      const table = base.table("tableName");
+      const updateSpy = sinon.fake();
```
Comment (alexquick) [inline src/ChangeDetector.test.js:21]: Nice!

File: src/ChangeDetector.js (hunk 1 above)
Comment (mjmaurer) [reply src/ChangeDetector.js:90]: agree its more readable

**Pull request merged**